modules/rosa-create-objects.adoc

```diff
@@ -140,6 +140,9 @@ a|--cluster-name <cluster_name>
 |--dry-run
 |Simulates creating the cluster.
 
+|--domain-prefix
+|Optional. When used with the `create cluster` command, this argument sets the subdomain for your cluster on `openshiftapps.com`. The value for this argument must be unique within your organization, cannot be longer than 15 characters, and cannot be changed after cluster creation. 
```

Comment:
We could additionally mention that if not supplied, an autogenerated value will be used; either the cluster's name could be used if it's less than or equal to 15 chars and not already used otherwise it'll be randomly generated to a 15 chars string.